instrumentation/opentelemetry-annotations-1.0/javaagent/opentelemetry-annotations-1.0-javaagent.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
   compileOnly project(path: ':opentelemetry-ext-annotations-shaded-for-instrumenting', configuration: 'shadow')
 
   testImplementation deps.opentelemetryExtAnnotations
+  testImplementation "net.bytebuddy:byte-buddy-gradle-plugin:${versions.bytebuddy}"
```

Comment:
Why gradle plugin and not just byte buddy?

Reply:
copy-pasted it without paying attention, fixed.